converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java

```diff
@@ -114,6 +114,11 @@ private JsonWrapper encodeMeasurementSets(Map<TemplateId, Node> childMapByTempla
 
 				sectionEncoder.encode(childWrapper, child);
 				childWrapper.put("source", "qrda3");
+				if (TemplateId.MEASURE_SECTION_V4.getRoot().equalsIgnoreCase(childType.getRoot())
+					&& ClinicalDocumentDecoder.MIPS_APM.equalsIgnoreCase(
+						currentNode.getValue(ClinicalDocumentDecoder.RAW_PROGRAM_NAME))) {
+					childWrapper.put(ClinicalDocumentDecoder.PROGRAM_NAME, ClinicalDocumentDecoder.MIPS.toLowerCase());
```

Comment:
I think this one needs: `ClinicalDocumentDecoder.MIPS.toLowerCase(Locale.getDefault())` its failing with `[ERROR] Use of non-localized String.toLowerCase() `

Reply:
I missed this after all the develop branch merges. Thanks for pointing it out. Got it fixed and build runs locally.